src/main/java/net/wasdev/wlp/common/plugins/util/InstallFeatureUtil.java

```diff
@@ -709,55 +711,49 @@ private void productInfoValidate() throws PluginExecutionException {
      * Runs the productInfo command and returns the output
      * 
      * @param installDirectory The directory of the installed runtime
-     * @param action The action to perform for the productInfo command
+     * @param action           The action to perform for the productInfo command
      * @return The command output
      * @throws PluginExecutionException if the exit value of the command was not 0
      */
-    public static String productInfo(File installDirectory, String action) throws PluginExecutionException {
+    private String productInfo(File installDirectory, String action) throws PluginExecutionException {
         Process pr = null;
-        InputStream is = null;
-        Scanner s = null;
-        Worker worker = null;
+        BufferedReader in = null;
+        StringBuilder sb = new StringBuilder();
         try {
-            String command;
+            String productInfoFile;
             if (OSUtil.isWindows()) {
-                command = installDirectory + "\\bin\\productInfo.bat " + action;
+                productInfoFile = installDirectory + "\\bin\\productInfo.bat";
             } else {
-                command = installDirectory + "/bin/productInfo " + action;
+                productInfoFile = installDirectory + "/bin/productInfo";
             }
-            pr = Runtime.getRuntime().exec(command);
-            worker = new Worker(pr);
-            worker.start();
-            worker.join(300000);
-            if (worker.exit == null) {
+            ProcessBuilder pb = new ProcessBuilder(productInfoFile, action);
+            pr = pb.start();
+
+            in = new BufferedReader(new InputStreamReader(pr.getInputStream()));
+            String line;
+            while ((line = in.readLine()) != null) {
+                sb.append(line);
```

Comment:
Use `sb.append(line).append(System.lineSeparator());` so that the result includes line breaks, since it will be displayed to the user if there is a validation error.